CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Add events `meetingReconnected`, `signalingDropped` and `receivingAudioDropped` to `eventDidReceive` by publishing them as stand alone events. Currently, these events were only included in the meeting history attribute when a meeting event is published. 
 - Added support for skipping full SDP renegotiations when switching simulcast streams.  This will result in less freezing when switching between layers in response to a network event as done in `VideoPriorityBasedPolicy`.  This will have no impact if not using simulcast.
 - Add link to SIP Media Application examples in README.
+- Add API `isSimulcastSupported` so applications can check whether simulcast can be enabled and pass corresponding 
```

Comment:
Seems like we are 50/50 on whether to manually split these onto new lines :P.

Reply:
I always used to `qq` split them, but one-line-per-changelog works fine, so let's go with that.